docs/nextflow_schema/nextflow_schema_examples.md

```diff
@@ -9,12 +9,12 @@ hide:
 You can see an example JSON Schema for a Nextflow pipeline `nextflow_schema.json` file below.
 
 This file was generated from the nf-core pipeline template, using `nf-core create`.
-It is used as a test fixture in the nf-validation package [here](https://github.com/nextflow-io/nf-validation/blob/master/plugins/nf-validation/src/testResources/nextflow_schema.json).
+It is used as a test fixture in the nf-schemakage [here](https://github.com/nextflow-io/nf-snf-schemaster/plugins/nf-schemnf-schemaources/nextflow_schema.json).
```

Comment:
This looks like it borked itself with multiple replaces inside itself?

Reply:
I'm not even sure how that happened... But I'll fix it!